decentraland/redux/actions.js

```diff
@@ -20,10 +23,19 @@ export function setAccountCreationStatus(creationStatus) {
   return { type: SET_ACCOUNT_CREATION_STATUS, creationStatus };
 }
 
+// deprecated
 export function setAccountCreationCurrentAction(creationCurrentAction) {
   return { type: SET_ACCOUNT_CREATION_CURRENT_ACTION, creationCurrentAction };
 }
 
+export function updateAccountCreationStepActionId(status, actionId) {
```

Comment:
Is this for updating a single action or updating the list of actions?

Reply:
This code is outdated, the action function that is using now is `updateAccountCreationCurrentStatusAction(action)`

Reply:
Ah ok yep - I accidentally viewed the UI in such a way that I wasn't able to review a subset of the latest commits together, so I thought this might be a risk. Do you know of a way to view a few commits together when the option to view the changes since you last views disappears?

Reply:
No, I don't.

Reply:
Okay, oh well. Thanks. Resolving this.